spec/requests/api/v3/providers/courses/index_spec.rb

```diff
@@ -51,187 +52,447 @@ def perform_request
+              "type" => "courses",
+              "attributes" => {
+                "findable?" => true,
+                "open_for_applications?" => true,
+                "has_vacancies?" => true,
+                "name" => provider.courses[0].name,
+                "course_code" => provider.courses[0].course_code,
+                "start_date" => provider.courses[0].start_date.strftime("%B %Y"),
+                "study_mode" => "full_time",
+                "qualification" => "pgce_with_qts",
+                "description" => "PGCE with QTS full time teaching apprenticeship",
+                "content_status" => "published",
+                "ucas_status" => "running",
+                "funding_type" => "apprenticeship",
+                "is_send?" => true,
+                "level" => "primary",
+                "applications_open_from" => provider.courses[0].applications_open_from.strftime("%Y-%m-%d"),
+                "about_course" => enrichment.about_course,
+                "course_length" => enrichment.course_length,
+                "fee_details" => enrichment.fee_details,
+                "fee_international" => enrichment.fee_international,
+                "fee_uk_eu" => enrichment.fee_uk_eu,
+                "financial_support" => enrichment.financial_support,
+                "how_school_placements_work" => enrichment.how_school_placements_work,
+                "interview_process" => enrichment.interview_process,
+                "other_requirements" => enrichment.other_requirements,
+                "personal_qualities" => enrichment.personal_qualities,
+                "required_qualifications" => enrichment.required_qualifications,
+                "salary_details" => enrichment.salary_details,
+                "last_published_at" => enrichment.last_published_timestamp_utc.iso8601,
+                "about_accrediting_body" => nil,
+                "english" => "must_have_qualification_at_application_time",
+                "maths" => "must_have_qualification_at_application_time",
+                  "science" => "must_have_qualification_at_application_time",
+                "provider_code" => provider.provider_code,
+                "recruitment_cycle_year" => current_year.to_s,
+                "gcse_subjects_required" => %w[maths english science],
+                "age_range_in_years" => provider.courses[0].age_range_in_years,
+                "accrediting_provider" => nil,
+                "accrediting_provider_code" => nil,
+              },
+              "relationships" => {
+                "accrediting_provider" => { "meta" => { "included" => false } },
+                "provider" => { "meta" => { "included" => false } },
+                "site_statuses" => { "meta" => { "included" => false } },
+                "sites" => { "meta" => { "included" => false } },
+                "subjects" => { "meta" => { "included" => false } },
+              },
+              "meta" => {
+                "edit_options" => {
```

Comment:
should these `edit_options` be exposed?

Reply:
They're unnecessary but metadata can't be individually exposed/hidden like other attributes because they're non-standard and I don't want to hardcode it to disable metadata if this is meant to be an open API. I think we should leave it for now.

Reply:
We'll need a custom serialiser anyway, so while it uses the v2 serialiser sounds like it's not worth the effort to remove it.